### PR TITLE
docs: correct hint's description of --strict flag

### PR DESCRIPTION
**Pull request by minhtaile2712**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

[Comment in Issue #2057](https://github.com/nestjs/nest-cli/issues/2057#issuecomment-1537117486)

## What is the new behavior?

[Invitation for PR](https://github.com/nestjs/nest-cli/issues/2057#issuecomment-1549083356)

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/introduction.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ $ npm i -g @nestjs/cli
 $ nest new project-name
 ```
 
-> info **Hint** To create a new project with TypeScript's [strict](https://www.typescriptlang.org/tsconfig#strict) mode enabled, pass the `--strict` flag to the `nest new` command. 
+> info **Hint** To create a new TypeScript project with stricter feature set, pass the `--strict` flag to the `nest new` command.
```
Comment (micalevisk) [inline content/introduction.md:26]:
```suggestion
> info **Hint** To create a new TypeScript project with stricter feature set, pass the `--strict` flag to the [`nest new`](https://docs.nestjs.com/cli/usages#nest-new) command.
```

**Review by micalevisk — APPROVED** [review]

**Comment by minhtaile2712** [comment]
@micalevisk Can you merge this pull request?

**Comment by micalevisk** [comment]
@minhtaile2712 it will be merged by Kamil when he can, don't worry :) thanks for your contribution.

**Comment by minhtaile2712** [comment]
@micalevisk Love this. Thank you!

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**